build/checkstyle/tachyon_checks.xml

```diff
@@ -47,6 +47,11 @@
     <property name="fileExtensions" value=".java"/>
   </module>
   <module name="RegexpSingleline">
+    <property name="format" value="TODO[^\(]"/>
```

Comment:
Actually I am pretty fine with the current regexp. 

I was just thinking it will better if we can restrict this regexp matching within the scope of comment. Is it possible? Otherwise globally matching and rejecting string  "TODO" seems a bit tricky to me.